fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/OperatorVisitor.java

```diff
@@ -323,4 +325,11 @@ public R visitPhysicalNoCTE(PhysicalNoCTEOperator node, C context) {
         return visitOperator(node, context);
     }
 
+    public R visitPhysicalStreamJoin(PhysicalStreamJoinOperator node, C context) {
+        return visitOperator(node, context);
+    }
+
+    public R visitPhysicalStreamAgg(PhysicalStreamAggOperator node, C context) {
```

Comment:
you add three operator type, but just two physical operator here, PHYSICAL_BINLOG_SOURCE is missed?

Reply:
fixed